EasyCommands.Tests/ScriptTests/MockEntityUtility.cs

```diff
@@ -3,12 +3,23 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using Moq;
 using Sandbox.ModAPI.Ingame;
+using Sandbox.ModAPI.Interfaces;
 using VRageMath;
 
 namespace EasyCommands.Tests.ScriptTests {
     class MockEntityUtility {
 
+        public static Mock<ITerminalAction> MockCalledAction<T>(Mock<T> mockBlock, String actionName) where T : class, IMyTerminalBlock {
+            Mock<ITerminalAction> terminalAction = new Mock<ITerminalAction>();
+            terminalAction.Setup(action => action.Apply(mockBlock.Object));
```

Comment:
Why call setup for the `Apply` method without a corresponding Returns or other behavior?

Reply:
I think it was failing before I added the setup.  I can re-check

Reply:
Nope! Not needed. removed